docs/project-page.adoc

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
Is this change actually related to the new feature? (it's really hard to see what's changed on that line).

Reply:
The change removes a `<p>` tag around the first sentence, but... I'm not sure why, the site seems to be fine as it is. Nevermind, I will remove the change.